src/main/webapp/registration.jsp

```diff
@@ -0,0 +1,195 @@
+<%@page import="com.model.RegistrationModel"%>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@page import="com.controller.RegistrationUserListController"%>
+<%@page import= "com.controller.AjaxHandler" %>
+<%@ page errorPage="error.jsp" %> 
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="ISO-8859-1">
+		<title>Employee Registration Form</title>
+		<link rel="stylesheet" href="css/bootstrap.css"></link>
+		<link rel="stylesheet" href="css/button.css"></link>
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js" ></script>
+		<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.0/jquery.min.js"> </script>
+		<script src="https://code.jquery.com/jquery-3.6.1.min.js"></script>
+		<script src="https://code.jquery.com/ui/1.13.2/jquery-ui.min.js" ></script>
+		<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-validate/1.19.0/jquery.validate.min.js"> </script>
+		<script src="https://kit.fontawesome.com/yourcode.js"></script>
+		<script src='https://kit.fontawesome.com/a076d05399.js'></script>
+		<script src="https://unpkg.com/sweetalert/dist/sweetalert.min.js"></script>
+		<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+		<style>
+			#frm {
+				width: 400px;
+				height: auto;
+				margin: auto;
+				margin-top: 50px;
+				margin-bottom: 20px;
+			}
+		</style>
+		<!-- Form validation using jQuery -->
+		<script>
+			$(document).ready(function() {
```

Comment:
Please mention all scripting functions into single script tag and move script tag at the last of HTML design